src/serlo-editor-integration/editor-response-to-state.ts

```diff
@@ -149,23 +150,15 @@ export function editorResponseToState(uuid: MainUuidType): DeserializeResult {
           meta_description,
         },
       },
-      converted: !isEdtr(convertEditorState(content) || empty),
     }
 
     function getContent() {
-      const convertdContent = convertEditorState(content)
-
-      const convertedContent = toEdtr(convertdContent) as
-        | RowsPlugin
-        | OtherPlugin
+      const convertedContent = toEdtr(convertEditorState(content))
```

Comment:
slightly confused what `convertEditorState` is doing: so it's a wrapper around JSON.parse? I would expect another name, e.g. parseState.

Reply:
Good idea, I changed it in https://github.com/serlo/frontend/pull/2891